runtime/shibuya/src/precompiles.rs

```diff
@@ -125,6 +145,16 @@ pub type ShibuyaPrecompilesSetAt<R, C> = (
         UnifiedAccountsPrecompile<R, UnifiedAccounts>,
         (CallableByContract, CallableByPrecompile),
     >,
+    PrecompileAt<
+        AddressU64<20487>,
+        DispatchLockdrop<
+            R,
+            DispatchFilterValidate<RuntimeCall, WhitelistedLockdropCalls>,
+            ConstU32<8>,
+        >,
+        // Not callable from smart contract nor precompiled, only EOA accounts
```

Comment:
That would also mean we don't need to register the revert code, if I'm not mistaken.

Reply:
Can you elaborate ?

Reply:
When a new precompile is added, and we want it to be accessible from the smart contracts, we need to add a revert op code to the precompile address. Otherwise any call to the precompile would fail.

Same doesn't apply if it's called directly via Ethereum transaction.

Reply:
Yes we don't need revert code here, it's only EOA accounts callabale